account.rs/src/wasm/src/account.rs

```diff
@@ -313,20 +421,83 @@ impl Account {
     }
 
     #[wasm_bindgen(js_name = getPublicKey)]
-
     pub fn get_public_key(&self) -> Vec<u8> {
-        Account::key_to_vec(self.public_key)
+        key_to_vec(self.public_key)
+    }
+
+    fn get_hashing_secret_string(&self) -> Result<String, AccountError> {
+        let hashing_secret = self.hashing_secret.as_ref().ok_or(AccountError::Generic(
+            "hashing_secret is undefined".to_string(),
+        ))?;
+        let hashing_secret_string: String = vec_to_string(hashing_secret);
+        Ok(hashing_secret_string)
+    }
+
+    #[wasm_bindgen(js_name = getCombinedPublicKey)]
+    pub fn get_combined_public_key(&self) -> String {
+        let combined_public_keys: [u8; 64] = self
+            .public_key
+            .iter()
+            .cloned()
+            .chain(self.encryption_public_key.iter().cloned())
+            .collect::<Vec<_>>()
+            .try_into()
+            .unwrap();
```

Comment:
Could we return a `JsError` here?

Well, I would be fine with keeping `unwrap()` only in case it's impossible to get an error there (feel free to convince me if that's the case), but otherwise I would return an error.

Reply:
Fixed, thanks!